conf/app.routes

```diff
@@ -1,6 +1,7 @@
 # microservice specific routes
 
 # API
+GET        /api/rules                                 @uk.gov.hmrc.leakdetection.controllers.ApiController.rules
```

Comment:
wondering if the /api/rules endpoint should just return rule definitions and maybe have a /api/rules/count or /api/rules/violations endpoint to returns rules + leak count?

Reply:
The plan is to extend this to:
/api/rules?team=:team
/api/rules/:ruleId
/api/rules/:ruleId?team=:team
In all cases we care about both the rule and the repository counts.
Happy to rename it to rule-summaries or something similar rather than just rules though?